site/hxapi/hotel/av/index.md

```diff
@@ -2,316 +2,384 @@
+        <Code>LHRPAT</Code>
+        <BookingURL>/v1/hotel/HPLHRPAT</BookingURL>
+        <MoreInfoURL>/v1/product/LHRPAT</MoreInfoURL>
+        <RequestFlags/>
+        <Price>130.00</Price>
+        <PriceWithSurcharge>132.60</PriceWithSurcharge>
+        <Adults>2</Adults>
+        <Children>1</Children>
+        <RoomCode>TRL</RoomCode>
+        <BoardBasis>RO</BoardBasis>
+        <ParkingDays>8</ParkingDays>
+        <NonDiscPrice>130.00</NonDiscPrice>
+        <Filter>
+            <hotel_arrival_parking_ppts>1</hotel_arrival_parking_ppts>
+            <hotel_return_parking_ppts>1</hotel_return_parking_ppts>
+            <day_use_only>0</day_use_only>
+        </Filter>
+    </Hotel>
+    <Hotel>
+       <Name>Sheraton Skyline with parking at the hotel</Name>
+       <Code>LHRSKA</Code>
+       <BookingURL>/v1/hotel/HPLHRSKA</BookingURL>
+       <MoreInfoURL>/v1/product/LHRSKA</MoreInfoURL>
+       <RequestFlags/>
+       <Price>134.00</Price>
+       <PriceWithSurcharge>136.68</PriceWithSurcharge>
+       <Adults>2</Adults>
+       <Children>1</Children>
+       <RoomCode>TRL</RoomCode>
+       <BoardBasis>RO</BoardBasis>
+       <ParkingDays>8</ParkingDays>
+       <NonDiscPrice>134.00</NonDiscPrice>
+       <Filter>
+           <day_use_only>0</day_use_only>
+       </Filter>
+   </Hotel>
+   <Pricing>
+        <CCardSurchargePercent>2.00</CCardSurchargePercent>
+        <CCardSurchargeMin>0.00</CCardSurchargeMin>
+        <CCardSurchargeMax>4.00</CCardSurchargeMax>
+        <DCardSurchargePercent>0.00</DCardSurchargePercent>
+        <DCardSurchargeMin>0.00</DCardSurchargeMin>
+        <DCardSurchargeMax>0.00</DCardSurchargeMax>
+        <WaiverValue>2.99</WaiverValue>
+    </Pricing>
+    <SepaID/>
+    <API_Header>
+        <Request>
+            <ABTANumber>YourABTANumber</ABTANumber>
+            <Password>YourPassword</Password>
+            <key>YourKey</key>
+            <token>YourToken</token>
+            <ArrivalDate>2017-12-01</ArrivalDate>
+            <Nights>1</Nights>
+            <RoomType>T21</RoomType>
+            <ParkingDays>8</ParkingDays>
+            <v>1</v>
+        </Request>
+    </API_Header>
+</API_Reply>
+```
 
-The credit card surcharge IS applied to the TotalPrice + the Cancellation Waiver (see below). To prevent the surcharge from exceeding certain boundaries, we have min and max thresholds. If the amount does not come between those two figures, you should use the relevant threshold value.
+### European Hotel Availability Request - Room with Parking
 
-Pseudo code
-```
-x = ((TotalPrice + CanxWaiver) / 100 ) * CCardSurchargePercent
-if( x < CCardSurchargeMin)
-  x = CCardSurchargeMin
-else if x > CCardSurchargeMax
-  x = CCardSurchargeMax
+```html
+https://api.holidayextras.co.uk/v1/hotel/MUC?ABTANumber=YourABTANumber&Password=YourPassword&key=YourKey&token=YourToken&ArrivalDate=2017-12-01&Nights=1&RoomType=D20&ParkingDays=8&System=ABG&lang=de
```

Comment:
I have deliberately included it here because I have stripped out so much DE content already. They had their own part of the site with German translations of the English, bespoke hotel docs in English and additional content in the UK hotel docs - 3 different places! This is a concession so it's clear. We do have partners integrating both for UK and DE so worth keeping it together, e.g. Trusted Parking. I'd like to keep a section within this doc for EU so it's clear :)